### PR TITLE
Update interactions.md, angular.md, and interactions-attributes.html

### PR DESCRIPTION
**Pull request by billyshaominwu**
Update docs to reflect changes to UI components for react, react-native, angular, vue
in conjunction with https://github.com/aws-amplify/amplify-js/pull/2121

### CODE REVIEW

File: js/interactions.md [hunk 1]
```diff
@@ -142,8 +142,28 @@ When using React, you can use *ChatBot* with following properties;
...
     onComplete={this.handleComplete.bind(this)}
     clearOnComplete={true}
+    conversations={false}
 />
 ```
+The `conversations` props turns voice conversation mode on/off.
+
+If needed, you can also pass `config` in the props to modify the silence detection parameters, like in this example:
+
+```html
```
Comment (elorzafe) [inline js/interactions.md]:
```suggestion
```jsx
```

File: js/interactions.md [hunk 2]
```diff
@@ -142,8 +142,28 @@ When using React, you can use *ChatBot* with following properties;
     welcomeMessage="Welcome, how can I help you today?"
     onComplete={this.handleComplete.bind(this)}
     clearOnComplete={true}
+    conversations={false}
```
Comment (elorzafe) [inline js/interactions.md]: `conversations` prop it sounds a little odd. Maybe `conversationEnabled` or `conversationModeOn`?

File: js/interactions.md [hunk 3]
```diff
@@ -142,8 +142,28 @@ When using React, you can use *ChatBot* with following properties;
...
+const customVoiceConfig = {
+    silenceDetectionConfig: {
+        time: 2000,
+        amplitude: 0.2
+    }   
+}
+
+<ChatBot
+    ...
+    config={customVoiceConfig}
```
Comment (elorzafe) [inline js/interactions.md]: This wont be available on react-native?

File: js/interactions.md (hunk 1 above)
Comment (billyshaominwu) [reply js/interactions.md]: good catch 👍

File: js/interactions.md (hunk 2 above)
Comment (billyshaominwu) [reply js/interactions.md]: good point, `conversationModeOn` is much more descriptive

File: js/interactions.md (hunk 3 above)
Comment (billyshaominwu) [reply js/interactions.md]: correct, it won't be available on react-native

**Comment by erinewt1982** [comment]
Duplicate of #222

File: js/angular.md [hunk 4]
```diff
@@ -213,20 +213,34 @@ onAlbumImageSelected( event ) {
 
 ### Interactions
 
-The `amplify-interactions` component provides you with a drop-in Chat component that supports three properties:
+The `amplify-interactions` component provides you with a drop-in Chat component that supports five properties:
```
[the remark this answers is not in the document]
Comment (billyshaominwu) [reply js/angular.md]: yep, will have seven properties total

File: js/angular.md [hunk 5]
```diff
@@ -216,20 +216,39 @@ onAlbumImageSelected( event ) {
...
+    silenceDetectionConfig: {
+        time: 2000,
+        amplitude: 0.2
+    }   
+}
+
+```
+5. `voiceEnabled`: Enables voice user input
+6. `textEnabled`: Enables text user input
+7. `conversationModeOn`: Turns voice conversation mode on/off.
```
Comment (haverchuck) [inline js/angular.md]: Might be worth noting the defaults.

File: theme/_includes/interactions-attributes.html [hunk 6]
```diff
@@ -29,6 +29,39 @@
...
+        <td class="doc-td" data-column="Required">no</td>
+      </tr>
+      <tr class="doc-tr">
+        <td class="doc-td" data-column="Attribute">textEnabled</td>
+        <td class="doc-td" data-column="Type">boolean</td>
+        <td class="doc-td" data-column="Description">turns user text input on/off</td>
+        <td class="doc-td" data-column="Default">true</td>
+        <td class="doc-td" data-column="Required">no</td>
+      </tr>
     </tbody>
```
Comment (haverchuck) [inline theme/_includes/interactions-attributes.html]: Be sure to test change in viewport size.

File: js/angular.md [hunk 7]
```diff
@@ -216,20 +216,39 @@ onAlbumImageSelected( event ) {
...
 
 ```html
 <amplify-interactions 
     bot="yourBotName" 
     clearComplete="true" 
-    (complete)="onBotComplete($event)"></amplify-interactions>
+    (complete)="onBotComplete($event)"
+    [conversationModeOn]="false"
+    [voiceConfig]="customVoiceConfig"
```
[the remark this answers is not in the document]
Comment (billyshaominwu) [reply js/angular.md]: yep 👍

File: js/interactions.md [hunk 8]
```diff
@@ -142,9 +142,34 @@ When using React, you can use *ChatBot* with following properties;
     welcomeMessage="Welcome, how can I help you today?"
     onComplete={this.handleComplete.bind(this)}
     clearOnComplete={true}
+    conversationModeOn={false}
 />
 ```
 
+By default, the ChatBot will allow for both voice and text interaction. You can turn off text interaction by passing prop `textEnabled={false}` or you can turn off voice interaction by passing prop `voiceEnabled={false}`.
```
[the remark this answers is not in the document]
Comment (billyshaominwu) [reply js/interactions.md]: 👍

File: js/interactions.md [hunk 9]
```diff
@@ -142,9 +142,34 @@ When using React, you can use *ChatBot* with following properties;
...
+
+The `conversationModeOn` props turns continuous conversation cycle mode on/off for voice interaction.
```
[the remark this answers is not in the document]
Comment (billyshaominwu) [reply js/interactions.md]: 👍

**Pull request merged**